src/main/webapp/app/lecture/lecture-update.component.ts

```diff
@@ -34,9 +43,14 @@ export class LectureUpdateComponent implements OnInit {
     domainCommandsDescription = [new KatexCommand()];
 
     // Icons
+    faQuestionCircle = faQuestionCircle;
     faSave = faSave;
+    faPuzzleProcess = faPuzzlePiece;
     faBan = faBan;
     faHandShakeAngle = faHandshakeAngle;
+    file: File;
```

Comment:
Please follow the order in components, split with a line-break: 
1. Input/Output
2. State/Attributes
3. Everything else (icons, readonly vars)